includes/html/forms/update-service-status.inc.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2014 Neil Lathwood <https://github.com/laf/ http://www.lathwood.co.uk/fa>
```

Comment:
Would prefer not to add legacy code.  Also, this is the wrong copyright.
See, app/Http/Controllers/PortController.php for an example

Reply:
Agreed.
I could modify the includes/html/forms/create-service.inc.php which also updates, to make it a little bit more generic and allow update of one variable without providing all of them (as it is required now). Would you prefer that ?

Reply:
Last commit is getting rid of the new ajax_form file, and makes the original "create_service" versatile. Can now handle updates of any field that comes in the request. I had to take care of the checkboxes that are not sent when not checked. 
Let me know which version you prefer.